test/cljam/t_pileup.clj

```diff
@@ -42,26 +42,28 @@
   (is (= (with-open [r (bam/reader test-sorted-bam-file)]
            (plp/pileup r "ref" nil))
          test-bam-pileup-ref))
-  (is (= (with-open [r (bam/reader test-sorted-bam-file)]
-           (plp/pileup r "ref" {:step 2 :n-threads 4}))
-         test-bam-pileup-ref))
-  (is (= (with-open [r (bam/reader test-sorted-bam-file)]
-           (plp/pileup r "ref" {:step 3 :n-threads 4}))
-         test-bam-pileup-ref))
-  (is (= (with-open [r (bam/reader test-sorted-bam-file)]
-           (plp/pileup r "ref" {:step 5 :n-threads 4}))
-         test-bam-pileup-ref))
-  (is (= (with-open [r (bam/reader test-sorted-bam-file)]
-           (plp/pileup r "ref" {:step 7 :n-threads 4}))
-         test-bam-pileup-ref))
-  (is (= (with-open [r (bam/reader test-sorted-bam-file)]
-           (plp/pileup r "ref" {:step 11 :n-threads 4}))
-         test-bam-pileup-ref))
-  (is (= (with-open [r (bam/reader test-sorted-bam-file)]
-           (plp/pileup r "ref" {:step 13 :n-threads 4}))
-         test-bam-pileup-ref))
-  (is (= (with-open [r (bam/reader test-sorted-bam-file)]
-           (plp/pileup r "ref" {:n-threads 4}) test-bam-pileup-ref)))
+  (doseq [n-threads [4 1]]
+    (is (= (with-open [r (bam/reader test-sorted-bam-file)]
+             (plp/pileup r "ref" {:step 2 :n-threads n-threads}))
+           test-bam-pileup-ref))
+    (is (= (with-open [r (bam/reader test-sorted-bam-file)]
+             (plp/pileup r "ref" {:step 3 :n-threads n-threads}))
+           test-bam-pileup-ref))
+    (is (= (with-open [r (bam/reader test-sorted-bam-file)]
+             (plp/pileup r "ref" {:step 5 :n-threads n-threads}))
+           test-bam-pileup-ref))
+    (is (= (with-open [r (bam/reader test-sorted-bam-file)]
+             (plp/pileup r "ref" {:step 7 :n-threads n-threads}))
+           test-bam-pileup-ref))
+    (is (= (with-open [r (bam/reader test-sorted-bam-file)]
+             (plp/pileup r "ref" {:step 11 :n-threads n-threads}))
+           test-bam-pileup-ref))
+    (is (= (with-open [r (bam/reader test-sorted-bam-file)]
+             (plp/pileup r "ref" {:step 13 :n-threads n-threads}))
+           test-bam-pileup-ref))
+    (is (= (with-open [r (bam/reader test-sorted-bam-file)]
+             (plp/pileup r "ref" {:n-threads n-threads})
+             test-bam-pileup-ref))))
```

Comment:
I'm sorry, the original test case was incorrect. `test-bam-pileup-ref` must be outside of `with-open`